src/AddressRegistry.Api.BackOffice.Handlers.Sqs.Lambda/Requests/SqsLambdaAddressApproveRequest.cs

```diff
@@ -0,0 +1,26 @@
+namespace AddressRegistry.Api.BackOffice.Handlers.Sqs.Lambda.Requests
+{
+    using Abstractions.Requests;
+    using StreetName;
+    using StreetName.Commands;
+    using IHasAddressPersistentLocalId = Abstractions.IHasAddressPersistentLocalId;
+
+    public sealed class SqsLambdaAddressApproveRequest :
+        SqsLambdaRequest,
+        IHasBackOfficeRequest<AddressBackOfficeApproveRequest>,
+        IHasAddressPersistentLocalId
+    {
+        public AddressBackOfficeApproveRequest Request { get; set; }
+
+        public int AddressPersistentLocalId => Request.PersistentLocalId;
+
+        /// <summary>
+        /// Map to ApproveAddress command
+        /// </summary>
+        /// <returns>ApproveAddress.</returns>
+        public ApproveAddress ToCommand(AddressPersistentLocalId addressPersistentLocalId)
```

Comment:
AddressPersistentLocalId should not be a paramater, but instead use the class property.